functions/ext-storage.bash

```diff
@@ -1,131 +1,110 @@
-  # exit if root is not on SDCARD
-  if ! [ $rootonsdcard = true ]; then
-    infotext="It seems as if your system root is not on the SD card.
-       ***Aborting, process cant be started***"
-    whiptail --title "System root not on SD card ?" --msgbox "$infotext" 8 78
+  # Exit if root is not on SD card
+  if [[ $rootOnSD != "true" ]]; then
+    whiptail --title "System root not on SD card!" --msgbox "It seems that your system root is not on the SD card.\\n\\n***Aborting, process cannot be started!***" 9 80
     return 0
   fi
 
-  # exit if destination is not available
-  if ! [[ -b "$NEWROOTPART" ]]; then
-    infotext="Seems there is no external storage medium inserted that we could move openHABian root to.\\n***Aborting, process cant be started***"
-    whiptail --title "No destination SD card ?" --msgbox "$infotext" 8 78
+  # Exit if destination is not available
+  if ! [[ -b "$newRootPart" ]]; then
+    whiptail --title "No destination device?" --msgbox "It seems that there is no external storage medium inserted that we could move openHABian root to.\\n\\n***Aborting, process cannot be started!***" 9 80
     return 0
   fi
 
-  srcsize=$(blockdev --getsize64 /dev/mmcblk0)
-  destsize=$(blockdev --getsize64 "$NEWROOTDEV")
-  if [[ "$srcsize" -gt "$destsize" ]]; then
-    infotext="Your internal storage medium is larger than the external device/medium you want to move your root to. This will very likely break your system.\\n\\nDo you still REALLY want to continue ?"
-    if ! (whiptail --title "Move system root to '$NEWROOTPART'" --yes-button "Continue" --no-button "Stop" --defaultno --yesno "$infotext" 12 78) then echo "CANCELED"; return 0; fi
+  if [[ $srcSize -gt $destSize ]]; then
+    if ! (whiptail --title "Destination device to small?" --yes-button "Continue" --no-button "Cancel" --defaultno --yesno "Your internal storage medium is larger than the external device/medium you want to move your root to. This will very likely break your system.\\n\\nDo you still REALLY want to continue?" 10 80); then echo "CANCELED"; return 0; fi
   fi
 
-  # check if USB power is already set to 1A, otherwise set it there
-  if grep -q -F 'max_usb_current=1' /boot/config.txt; then
-     echo "max_usb_current already set, ok"
-  else
-     echo 'max_usb_current=1' >> /boot/config.txt
-
-     echo
-     echo "********************************************************************************"
-     # shellcheck disable=SC2086
-     echo "REBOOT, run $(basename $0) again and recall menu item 'Move root to USB'"
-     echo "********************************************************************************"
-     whiptail --title "Reboot needed!" --msgbox "USB had to be set to high power (1A) first. Please REBOOT and RECALL this menu item." 15 78
-     exit 0
+  # Check if USB power is already set to 1A, otherwise set it here
+  if ! grep -qsF 'max_usb_current=1' /boot/config.txt; then
+    echo -n "$(timestamp) [openHABian] Setting max USB power draw to 1A... "
```

Comment:
more precisely it's unlimiting the power draw to up to 1.2A on older RPi
on newer (4, not sure of 3) it has no effect (but does not do harm either)

Reply:
Correct, I just left that comment alone and didn't update it but I will change it to reflect reality.